src/install.rst

```diff
@@ -48,9 +48,15 @@ These instructions will install the Nextstrain CLI and tools to run and view you
 
       1. Install Miniconda:
 
-         a. Go to the `installation page <https://docs.conda.io/en/latest/miniconda.html>`_.
-         b. Scroll down to the **Latest Miniconda Installer Links** section and click the MacOSX platform link that ends with **pkg**.
-         c. Open the downloaded file and follow through installation prompts.
+         .. The installer link is taken from https://docs.conda.io/en/latest/miniconda.html.
+
+         a. Download the ``.pkg`` installer by clicking `here <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_.
```

Comment:
Pushed b9aa8bafed63dcafb17d61bebd070e4deffeb326 to remove the "click here" wording.

Dropped mention of `.pkg` entirely since we're directly linking the thing we want folks to install.

Reply:
Thanks, makes sense to me!